python/tvm/topi/x86/dense.py

```diff
@@ -293,16 +324,13 @@ def dense_vnni_compute(cfg, X, packed_w, bias=None):
             ),
             axis=ak,
         ),
-        tag="dense_vnni",
-        attrs={"schedule_rule": "dense_vnni"},
```

Comment:
This shouldn't be removed, it is used here https://github.com/apache/tvm/blob/b16a64d6edb9fd1a014fc51995dff7d0e2f4c84e/tests/python/unittest/test_meta_schedule_vnni_integration.py#L198.

Since this only affects MetaSchedule, you don't have to provide this value for AMX. So only when `dense_int8_compute` is called for VNNI, you need to provide this attribute.

Reply:
@masahi Is this case only use the x86 int8 compute method and inject a particular TIR scheduling? Can we just change the attribute `dense_vnni` to `dense_int8` which used here?

Reply:
Yes, but it is important that we specify that we use this compute for VNNI. If the schedule rule annotation only says "dense_int8", we don't know which intrinsic to tensorize this compute.

Reply:
> This shouldn't be removed, it is used here
> 
> https://github.com/apache/tvm/blob/b16a64d6edb9fd1a014fc51995dff7d0e2f4c84e/tests/python/unittest/test_meta_schedule_vnni_integration.py#L198
> 
> .
> Since this only affects MetaSchedule, you don't have to provide this value for AMX. So only when `dense_int8_compute` is called for VNNI, you need to provide this attribute.

@masahi Sorry, may given the misunderstanding, I mean that can we use the `dense_int8` in this test case?  cuz here inject the VNNI intrinsic explicitly. 
And by default in relay build flow, it will check if the VNNI or AMX is availiable and chose different schedulling.
I've verified that this test case still functional after the little modification [in this commit](https://github.com/apache/tvm/pull/13642/commits/5718a059c69972cf71ea082a3303b5c29fa2d21f)  bellow

Reply:
Of course the test still works, because it was already written for VNNI. The point is that the name `dense_vnni` tells that the particular TE expression is meant for VNNI tensorization, in particular the weight is pre-packed appropriately. But a generic name like `dense_int8` doesn't provide such information. 

So please revert that commit and restore and pass `attrs={"schedule_rule": "meta_schedule.x86.dense_vnni"}` when you create a compute expression for VNNI. `schedule_rule` is not relevant for AMX for now.

Reply:
@masahi  yep, got it, the schedule rule for ms dense_vnni are restored.  
Keep it remained but AMX not use it for now. 
( A smalll question: How do we know inside of this compute expression if it's created for VNNI not AMX? )

Reply:
I suggest checking the target string in the op strategy, and create separate compute for VNNI or AMX (rather than using the same function, `dense_int8`, for both).